packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -99,26 +99,38 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
 
   const sandboxMode = process.env.NEXT_PUBLIC_IS_E2E ? true : false;
 
+  const pageSize = 90;
+  let pageNumber = 0;
+
   //delete batch_ids with already past scheduled date from scheduled_sends
-  const remindersToDelete = await prisma.workflowReminder.findMany({
-    where: {
-      method: WorkflowMethods.EMAIL,
-      cancelled: true,
-      scheduledDate: {
-        lte: dayjs().toISOString(),
+  while (true) {
+    const remindersToDelete = await prisma.workflowReminder.findMany({
+      where: {
+        method: WorkflowMethods.EMAIL,
+        cancelled: true,
+        scheduledDate: {
+          lte: dayjs().toISOString(),
+        },
       },
-    },
-  });
+      skip: pageNumber * pageSize,
```

Comment:
load workflowReminders in batches (we also do that in `api/crondowngradeUsers` and `api/cron/monthlyDigestEmail`)